build/Packages.targets

```diff
@@ -13,19 +13,26 @@
       https://dotnet.myget.org/F/project-system-interop/api/v3/index.json;
       https://vside.myget.org/F/vs-impl/api/v3/index.json;
       https://vside.myget.org/F/vssdk/api/v3/index.json;
+      https://api.nuget.org/v3/index.json;
     </RestoreSources>
   </PropertyGroup>
 
   <ItemGroup>
 
     <!-- Toolset -->
-    <PackageReference Update="RoslynTools.RepoToolset"                                                Version="$(RoslynToolsRepoToolsetVersion)" />
-    <PackageReference Update="RoslynTools.VsixExpInstaller"                                           Version="$(RoslynToolsVsixExpInstallerVersion)" />
+    <PackageReference Update="RoslynTools.RepoToolset"                                                Version="$(RoslynToolsRepoToolsetVersion)" />    
+    <PackageReference Update="RoslynTools.ModifyVsixManifest"                                         Version="$(RoslynToolsModifyVsixManifestVersion)" />
+    <PackageReference Update="RoslynTools.SignTool"                                                   Version="$(RoslynToolsSignToolVersion)" />
+    <PackageReference Update="XliffTasks"                                                             Version="$(XliffTasksVersion)" />
     <PackageReference Update="RoslynDependencies.ProjectSystem.OptimizationData"                      Version="$(RoslynDependenciesProjectSystemOptimizationDataVersion)" />
     <PackageReference Update="Microsoft.DotNet.IBCMerge"                                              Version="[$(MicrosoftDotNetIBCMergeVersion)]" />
     <PackageReference Update="Microsoft.DiaSymReader.Pdb2Pdb"                                         Version="$(MicrosoftDiaSymReaderPdb2PdbVersion)" />
+    <PackageReference Update="Microsoft.Net.Compilers"                                                Version="3.3.0-beta2-19357-02" />
     <PackageReference Update="OpenCover"                                                              Version="$(OpenCoverVersion)" />
     <PackageReference Update="Codecov"                                                                Version="$(CodecovVersion)" />
+    <PackageReference Update="MicroBuild.Core"                                                        Version="$(MicroBuildCoreVersion)" />
```

Comment:
Why do we need to update these?

Reply:
See the update, I've explicitly set their version. I want to remove the double-restore - so I've moved them into our dependency graph.

Reply:
All the micro-build package should be restore-able only on a signed build machine. I don't understand how setting the version of packages that will fail to restore / aren't restored fixes things

Reply:
> All the micro-build package should be restore-able only on a signed build machine
>  I don't understand how setting the version of packages that will fail to restore

Explain what you mean by this. This doesn't fail to restore.

Reply:
I see, `MicroBuild.Core` was uploaded to the dotnet-buildtools myget feed [here](https://dotnet.myget.org/feed/dotnet-buildtools/package/nuget/MicroBuild.Core). So restore will work. I think this won't break the signed build (which uses version 1.42) but you should check.

Reply:
And now that I checked on the signed build, it looks like that was broken in a recent PR...

Reply:
This one: https://github.com/dotnet/project-system/pull/5107/files would be one cause @drewnoakes.

Reply:
See: %USERPROFILE\.nuget\packages\roslyntools.repotoolset\1.0.0-beta-62705-01\tools\Tools.proj

We have these dependencies, we use it to build the Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles project.

Reply:
@jmarolf Can we modify .vsts-ci.yml to call something new, or does this require us to push that to every branch?

Reply:
Jon answered with yes, .vsts-ci.yml is per branch.